### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by therajatg**
Hi! need code my alert component to be reviewed.

### CODE REVIEW

File: styles/component-styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.error{
+    background-color: #ff4040;
+    padding: 1rem;
+    border-radius: 1rem;
+    border: 2px solid darkred;
+}
+
+.warning{
+    background-color: #fff44f;
```
Comment (Tanjum786) [inline styles/component-styles/alert.css]: Use colors in variables. It will help you to reuse

File: styles/component-styles/alert.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.alert-container{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+}
+
+.error{
```
Comment (shashidwivedi) [inline styles/component-styles/alert.css:21]: The generic names like error, warning, success, info, etc. can cause confusion while maintaining the CSS. You can give a prefix, preferably the component these variations belongs to. In this case, the alert component

File: styles/component-styles/alert.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/alert.css]: Oh! yes, I forgot to use variables. Thanks for pointing it out. rectified!

File: styles/component-styles/alert.css (hunk 2 above)
Comment (therajatg) [reply styles/component-styles/alert.css:21]: Good thinking Shashi. 
Added alert as the prefix in all 4 names.

Thanks

**Pull request merged**